fabtests/unit/cq_test.c

```diff
@@ -41,6 +41,8 @@
 #include "unit_common.h"
 #include "shared.h"
 
+#define TEST_MAX (1 << 15)
```

Comment:
This would be better as a command line test option, which defaults to 64k.

Reply:
Updated